test/output/fixable.json

```diff
@@ -1,5 +1,5 @@
 {
-    "basePath": "/v1",
+    "basePath": "/v1/",
```

Comment:
that was a lazy "fix". Let me revert back and see what's going on. I'm running `iojs` locally and had to make this change to make test pass. But now in travis, only `iojs` is passing!